.yamllint

```diff
@@ -2,6 +2,9 @@ extends: default
 ignore: |
   *.molecule/
   molecule/cookiecutter/
+  .tox
+  # HACK: https://github.com/pyupio/pyup/issues/346
+  .pyup.yml
```

Comment:
PYUP is the only tool that seems to choke on "hardened" yaml files, so we skip it.

Reply:
@ssbarnea you can add a separate lint once you finish pre-commit one